pulsar-client-api/src/main/java/org/apache/pulsar/client/api/schema/SchemaDefinitionBuilder.java

```diff
@@ -80,6 +80,15 @@
      */
     SchemaDefinitionBuilder<T> withPojo(Class pojo);
 
+    /**
+     * Set schema of pojo classLoader.
+     *
+     * @param classLoader pojo classLoader
+     *
+     * @return schema definition builder
+     */
+    SchemaDefinitionBuilder<T> withClassLoader(ClassLoader classLoader);
```

Comment:
If need to add a public API to fix the issue, it should start with a proposal first.